_resources/port1.0/group/legacysupport-1.1.tcl

```diff
@@ -175,7 +176,7 @@ proc legacysupport::add_legacysupport {} {
         set ls_cache_cppflags "[legacysupport::get_cpp_flags]"
 
         # Flags for using MP libcxx
-        if { [option legacysupport.use_mp_libcxx] } {
+        if { [option legacysupport.use_mp_libcxx] && ${configure.cxx_stdlib} eq "libc++" } {
```

Comment:
you may put `global configure.cxx_stdlib` just before this `if`

Reply:
@catap Done.